libs/auth-api-lib/src/lib/delegations/admin/delegation-admin-custom.service.ts

```
@@ -285,4 +295,90 @@
+      })
+    }
+  }
+
+  private formatZendeskDate(date: string | null): Date | null {
+    if (!date) {
+      return null
+    }
+
+    const [day, month, year] = date.split('.').map(Number)
+
+    if (!day || !month || !year || isNaN(day) || isNaN(month) || isNaN(year)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)